pytrack/core/parameter.py

```diff
@@ -9,47 +9,180 @@
 Description: PyTrack parameter
 """
 
+from pytrack.utils import is_jsonable
+from pytrack.core.data_classes import DVCParams
+from pathlib import Path
+from typing import Union
+
+
+class ParameterHandler:
+    def __init__(self):
```

Comment:
class doc string.

Reply:
solved in 2175398a95c5813644aef5da0d41bc8ce2df9d21